src/java.base/share/classes/jdk/internal/loader/URLClassPath.java

```diff
@@ -843,8 +847,8 @@ public void close() throws IOException {
          * a JAR file.
          */
         private JarLoader(URL url, URLStreamHandler jarHandler,
-                  HashMap<String, Loader> loaderMap,
-                  AccessControlContext acc, boolean usingSharedClasses)   //OpenJ9-shared_classes_misc
+        HashMap<String, Loader> loaderMap,
```

Comment:
This doesn't look right, for these two lines the formatting changed from the OpenJDK version.

Reply:
I recall this line was marked as conflicts as well along with next line.
Probably removed the tab space, will fix it.

Reply:
Initial `OpenJ9-shared_classes_misc` change also included adding more spaces at beginning of `HashMap<String, Loader> loaderMap,` which has been restored to OpenJDK version now.
Please have another look.